projects/components/src/dropdown/dropdown.component.html

```diff
@@ -0,0 +1,54 @@
+<clr-dropdown class="nested-dropdown">
+    <button
+        [ngClass]="{ 'btn btn-sm btn-link': dropdownTriggerBtnTxt }"
```

Comment:
What does it mean/how does it act if dropdownTriggerBtnTxt is unset?

Reply:
Those 3 classes will not be applied to the button element when that expression evaluates to false.